core/indigo-core/molecule/src/molfile_saver.cpp

```diff
@@ -149,6 +149,7 @@ void MolfileSaver::_saveMolecule(BaseMolecule& mol, bool query)
 
     MoleculeCIPCalculator mcc;
     mcc.updateCIPStereoDescriptors(mol, add_stereo_desc);
```

Comment:
change the function to be more obvious
if (add_stereo_desc) {
MoleculeCIPCalculator mcc;
mcc.addCIP....()
mcc.addCIPSgroups(mol);
}

Reply:
fixed